integration-tests/__tests__/globals.test.js

```diff
@@ -120,10 +120,10 @@ test('tests with no implementation', () => {
 
   writeFiles(TEST_DIR, {[filename]: content});
   const {stderr, status} = runJest(DIR);
-  expect(status).toBe(0);
+  expect(status).toBe(1);
 
   const {summary, rest} = extractSummary(stderr);
-  expect(rest).toMatchSnapshot();
+  // expect(rest).toMatchSnapshot();
```

Comment:
why are these commented out?

Reply:
I think this PR is pretty good to go other than that 🙂

Reply:
Ok, great. Let me make those changes and I'll push it up.

Reply:
@SimenB , that snapshot was commented out because it listed the absolute path to the file on my computer, which won't match when the CI is run. Any ideas on how to hide the path? 
<img width="777" alt="screen shot 2018-02-14 at 4 21 35 pm" src="https://user-images.githubusercontent.com/23248886/36228779-31df7260-11a3-11e8-9699-c0c3dfe9be3a.png">

Reply:
Your path is messed up - it's a relative path to an absolute one...